aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -199,6 +215,22 @@ void raw_cudnn_convolution_forward_out(
   // TODO: compile empty & fill_ using full_like or full
   Tensor requantize_multiplier_tensor = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast);
   requantize_multiplier_tensor.fill_(requantize_multiplier);
+  c10::optional<at::Tensor> bias_multiplier_tensor;
+  c10::optional<at::Tensor> after_scales_bias;
+  c10::optional<at::Tensor> after_add;
+  c10::optional<at::Tensor> broadcasted_bias;
+  if (bias.has_value()) {
+    std::vector<int64_t> new_size(quantized_output.sizes().size() - 1, 1);
```

Comment:
`quantized_output.sizes().size()` is the same as `quantized_output.ndim()` I think. 

also can we add some comment describing the broadcasting process as well, e.g. why we are using size() - 1 here etc.